data/io.elementary.desktop.wingpanel.datetime.gschema.xml

```diff
@@ -1,35 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schemalist>
-  <enum id="io.elementary.desktop.wingpanel.datetime.clock-formats">
-    <value nick="12h" value="0" />
-    <value nick="24h" value="1" />
-    <value nick="ISO8601" value="2" />
-  </enum>
     <schema path="/io/elementary/desktop/wingpanel/datetime/" id="io.elementary.desktop.wingpanel.datetime">
         <key type="b" name="show-weeks">
             <default>false</default>
             <summary>Show week numbers.</summary>
             <description>If this is set to true, week numbers will be shown in the calendar.</description>
         </key>
-        <key name="clock-format" enum="io.elementary.desktop.wingpanel.datetime.clock-formats">
```

Comment:
Deleting this key breaks Granite.DateTime.is_clock_format_12h.

Reply:
Yes, it should be merged with https://github.com/elementary/granite/pull/273